install_files/ansible-base/staging-specific.yml

```diff
@@ -46,11 +46,3 @@ sasl_username: "test"
 sasl_domain: "ossec.test"
 sasl_password: "password123"
 
-### Use for backup restores ###
-# If the `backup_zip` variable is defined ansible will copy the defined file to
-# the app server and run the 0.3_collect.py script to unzip and restore those
-# files.
-# ex: backup_zip: "sd-backup-2015-01-15--21-03-32.zip"
-# To use uncomment the following line and enter the correct decrypted zip
-# filename between the quotes.
-#backup_zip: ""
```

Comment:
@conorsch Yeah. Obviously all of this config management needs an overhaul anyway but this seemed like a reasonable short-term change.